template-home.php

```diff
@@ -2,30 +2,42 @@
 /*
 Template Name: Home
 */
+
+$home_info_bar = get_field( 'home_info_bar' );
+
+if( $home_info_bar ) :
```

Comment:
This will only check for null... a `! empty()` check is needed to make sure it's not an empty string.

Reply:
It worked without the empty, but I added it in any way.